dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml

```diff
@@ -0,0 +1,928 @@
+apiVersion: v1
+data:
+  grafana-dashboard.json: |
```

Comment:
I suggest renaming it to something to reflect the dashboard, maybe `konflux-argocd-dashboard.json`

Reply:
changed